### PR TITLE
Feature/test zktls demo

### PR DESCRIPTION
**Pull request by rsoury**

### CODE REVIEW

File: zk/tests/zktls/methods/guest/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+name = "verity_zk_tests_zktls_guest"
+version = "0.1.0"
+edition = "2021"
+
+[workspace]
+
+[dependencies]
+risc0-zkvm = { version = "1.1.2", default-features = false, features = ['std'] }
+verity-local-verify = { path = "../../../../../rs/local-verify" }
+verity-verifier = { path = "../../../../../../verity/verifier", default-features = false, features = [
```
Comment (rsoury) [inline zk/tests/zktls/methods/guest/Cargo.toml:11]: @victorshevtsov - `verity-verifier` across the zkVM and zkTLS demos - not sure if it's referencing `verity` submodule.

Comment (victorshevtsov) [reply zk/tests/zktls/methods/guest/Cargo.toml:11]: Yes, It definitely references the verity submodule.

File: zk/tests/zkvm/methods/Cargo.toml [hunk 2]
```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [build-dependencies]
-risc0-build = { workspace = true }
+risc0-build = { version = "1.1.2" }
```
Comment (victorshevtsov) [inline zk/tests/zkvm/methods/Cargo.toml:7]: Is there a reason to not update the version at the workspace level and inherit it here?

Comment (rsoury) [reply zk/tests/zkvm/methods/Cargo.toml:7]: Simply adopted this from RiscZero

**Pull request merged**